requirements-test.txt

```diff
@@ -2,5 +2,5 @@ psycopg2
 djangorestframework>=3.3
 coverage==3.7.1  # rq.filter: >=3,<4
 coveralls
-django-filter>=0.15,<1.0
+django-filter
```

Comment:
Yes, I know.. but it helps with being notified.
I can see your point though, so feel free to pin it.
(As for myself as a PR submitter, in case I notice that it fails because of something minor, I often just create another PR to fix it, too)

As for pytest-django we also test against Django master, but that job is allowed to fail: when there are several jobs you can see what the cause of the failure is more easily.
So an option might also be to test against pinned and unpinned versions of django-filter (possibly via #123).